### PR TITLE
[SMALLFIX]Supply the variable name to Preconditions.checkNotNull

### PR DESCRIPTION
**Pull request by lucy233**

### CODE REVIEW

File: core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java [hunk 1]
```diff
@@ -640,11 +640,11 @@ private void freeSpaceInternal(long sessionId, long availableBytes, BlockStoreLo
     }
     // 2. transfer blocks among tiers.
     // 2.1. group blocks move plan by the destination tier.
-    Map<String, Set<BlockTransferInfo>> blocksGroupedByDestTier = new HashMap<>();
+    Map<String, Set> blocksGroupedByDestTier = new HashMap<>();
```
Comment (apc999) [inline core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java]: Changing `Map<String, Set<BlockTransferInfo>>` to ` Map<String, Set>` will make Java compiler confused a bit in later reference of `blocksGroupedByDestTier` to justify the type of element, creating warnings.
Can you revert the change of this line?

**Comment by apc999** [comment]
@lucy233  can you take a look at my comment and address it?

**Comment by yupeng9** [comment]
Closing this due to inactivity. @lucy233 feel free to reopen when you have the comment addressed.

**Comment by lucy233** [comment]
reopen

**Pull request closed without merging**